projects/Unit/APIApproval.Approve.verified.txt

```diff
@@ -259,6 +259,7 @@ namespace RabbitMQ.Client
     public interface IBasicPublishBatch
     {
         void Add(string exchange, string routingKey, bool mandatory, RabbitMQ.Client.IBasicProperties properties, byte[] body);
+        void Add(string exchange, string routingKey, bool mandatory, RabbitMQ.Client.IBasicProperties properties, System.ReadOnlyMemory<byte> body);
```

Comment:
Unfortunately since this is a change to an interface, this means this is a breaking change. If you're wanting to get this in to a release before 7.0, you can't do this.

Reply:
I think this should go into 7.0 which we can release fairly soon. @lukebakken WDYT?

Reply:
As far as I know there are quite a few things that have been discussed for 7.0, one of which is a fully-async client. I think @stebet has done some work on that.

If "fairly soon" means within 6 months... maybe???

Reply:
Yeah, I've done some research and experiments. It'd probably require quite a lot of changes to the public interface, all of which are good i.m.o, like departing from Event based notifications (can be replaced with a pluggable interface), but I'll write up a proposal soon for that. Reverting any interface changes for now :)